src/main/steps/sequence.ts

```diff
@@ -314,6 +323,10 @@ export const sequence: Step[] = [
     showInSection: Sections.Costs,
     getNextStep: () => UPLOAD_YOUR_DOCUMENTS,
   },
+  {
+    url: JURISDICTION_CONNECTION_SUMMARY,
+    getNextStep: () => CHECK_ANSWERS_URL,
```

Comment:
Shouldn't this go to the `CERTIFICATE_NAME` after this question?

Reply:
the next page is your name, but i believe that hasnt been implemented yet as we did the story refinement for it today